apps/devhub/utils.py

```diff
@@ -10,92 +10,95 @@
 from amo.urlresolvers import linkify_escape
 from files.models import File, FileUpload
 from files.utils import parse_addon
+from validator.constants import SIGNING_SEVERITIES
 from validator.version import Version
 from . import tasks
 
 
-# FIXME: Move to `validator.constants`.
-SEVERITIES = {'trivial', 'low', 'medium', 'high'}
+def process_validation(validation, is_compatibility=False):
```

Comment:
I made the classic mistake of reading over this function so I could add a docstring and comments about what it did. So, now it's three separate functions, and a `process_validation` function that calls them all as appropriate, but I also simplified most of the old code in the process.